SPEC.md

```diff
@@ -25,29 +25,31 @@ To achieve these goals this specification is split out into a number of smaller
 
 To provide context to the specs outlined below we will walk through an example.
 
-A user wants to launch a container running two processes.
-The two processes the user wants to run are the apps named `example.com/reduce-worker-register` and `example.com/reduce-worker`.
+A user wants to launch a container running three processes.
+The three processes the user wants to run are the apps named `example.com/reduce-worker-register`, `example.com/reduce-worker`, and `example.com/reduce-backup`.
 First, the executor will check the cache and find that it doesn't have images available for these apps.
 So, it will make an HTTPS request to example.com and using the `<meta>` tags there finds that the containers can be found at:
 
-	https://storage-mirror.example.com/reduce-worker-register.aci
 	https://storage-mirror.example.com/reduce-worker.aci
+	https://storage-mirror.example.com/worker-backup.aci
+	https://storage-mirror.example.com/reduce-worker-register.aci
 
-The executor downloads these two images and puts them into its local on-disk cache.
-Then the executor extracts two fresh copies of the images to create instances of the "on-disk app format" and reads the two image manifests to figure out what binaries will need to be executed.
+The executor downloads these three images and puts them into its local on-disk cache.
+Then the executor extracts three fresh copies of the images to create instances of the "on-disk app format" and reads the three image manifests to figure out what binaries will need to be executed.
 
-Based on user input the executor now sets up the necessary cgroups, network interfaces, etc and forks the `register` and `reduce-worker` processes in their shared namespaces inside the container.
+Based on user input, the executor now sets up the necessary cgroups, network interfaces, etc. and runs the `pre-start` event handlers for each app.
+Next, it forks the `reduce-worker`, `worker-backup`, and `register` processes in their shared namespaces inside the container.
```

Comment:
could you add a note about "pivoted into their respective root filesystems" or similar?